pkg/exchange/bybit/convert.go

```diff
@@ -169,15 +189,44 @@ func toGlobalOrderStatus(status bybitapi.OrderStatus) (types.OrderStatus, error)
 	}
 }
 
-func hashStringID(s string) uint64 {
-	h := fnv.New64a()
-	h.Write([]byte(s))
-	return h.Sum64()
-}
+// processQuantity converts the quantity unit from quote coin to base coin if the order is a **MARKET BUY**.
```

Comment:
What do you think renaming `processQuantity` to `processMarketBuyQuantity`?

Reply:
ok